backend-server/collections/games.js

```diff
@@ -0,0 +1,59 @@
+const {suspects} = require("./suspects");
+const {Player} = require("../models/player");
+const {Game} = require("../models/game");
+
+const games = []
+exports.games = games;
+
+exports.addGame = (attributes) => {
```

Comment:
I may be doing something wrong, but I have not been able to start a second game instance. Doing `post('http://localhost:PORT/games', data={'name': 'username'})` and
`post('http://localhost:PORT/games', data={'name': 'newUsername'})` both return a game with an id of 1